tbx/project_styleguide/templates/patterns/pages/blog/blog_detail.html

```diff
@@ -23,35 +23,52 @@
 {% endblock %}
 
 {% block content %}
-    {% include "patterns/molecules/title-block/title-block.html" with item=page tags=False %}
-    {% pageurl page.blog_index as blog_index_url %}
+    <div class="grid grid--spacer-large streamfield">
+        <div class="authors">
+            {% for page_author in page.authors.all %}
+                {% include "patterns/molecules/author/author.html" with author=page_author.author item=page %}
+            {% endfor %}
+        </div>
 
-    {% for page_author in page.authors.all %}
-        {% include "patterns/molecules/author/author.html" with author=page_author.author %}
-    {% endfor %}
+        <div class="streamfield__title">
+            <h1 class="heading heading--one">{{ page.title }}</h1>
 
-    <div>
-        {% for tag in page.services %}
-            {% include "patterns/atoms/tag/tag.html" with item=tag small=True tag_link_base=blog_index_url %}
-        {% endfor %}
-    </div>
+            {% pageurl page.blog_index as blog_index_url %}
 
-    <div>
-        <time datetime="{{ item.date|date:'Y-m-d' }}">{{ page.date }}</time>
+            <div class="tags tags--with-title">
+                <span class="sr-only">Filter blog posts by category</span>
+                {% for tag in page.services %}
+                    {% include "patterns/atoms/tag/tag.html" with item=tag small=True tag_link_base=blog_index_url %}
+                {% endfor %}
+            </div>
+        </div>
 
-        {% if page.read_time %}
-            / {{ page.read_time }} min read
-        {% endif %}
-    </div>
+        <div class="blog-meta">
+            <span>Insight</span>
```

Comment:
There was a lot of discussion around the use of 'Insights' for blog pages - I don't think it was finally decided. I noticed that Jeth queried if this was even needed in the design. Might be best to remove it.

Reply:
Removed in https://github.com/torchbox/torchbox.com/pull/41/commits/43d701ba3377531ee43ff0bc195ac26e0c9e38db. 

(currently means that `page_type` doesn't output anything on this branch but can see you've updated on https://github.com/torchbox/torchbox.com/pull/40/files#diff-7df244d38c599e323ae9fcdba40adb28c16115df2d60e576a6ca22ee612eec8fL15